metatlas/tools/atlas_prefilter.py

```diff
@@ -117,20 +117,28 @@ def score_ms2_data(ms2_data: pd.DataFrame, aligned_atlas_df: pd.DataFrame,
     ms2_data_refs_merge['score'] = ms2_data_refs_merge['mms_out'].apply(lambda x: x['score'])
     ms2_data_refs_merge['matches'] = ms2_data_refs_merge['mms_out'].apply(lambda x: x['matches'])
 
+    ms2_data_refs_merge['ref_frags'] = ms2_data_refs_merge['spectrum_y'].apply(lambda x: len(x[0]) if x.any() else 0)
+    ms2_data_refs_merge['data_frags'] = ms2_data_refs_merge['spectrum_x'].apply(lambda x: len(x[0]) if x.any() else 0)
+    ms2_data_refs_merge['reference_data_ratio'] = ms2_data_refs_merge['ref_frags']/ms2_data_refs_merge['data_frags']
+    ms2_data_refs_merge['match_reference_ratio'] = ms2_data_refs_merge['matches']/ms2_data_refs_merge['ref_frags']
+
+    ms2_data_refs_merge.to_csv("/out/ms2_data_refs_merge.csv")
```

Comment:
We should remove this diagnostic csv saving line before merging into main.

Reply:
Ha, thanks for catching that. Looking at the error this afternoon.

Reply:
Also, I'm going to keep the other ratio just for reference (reference_data_ratio) even though it's not used for now, but I'll drop the intermediate counting columns (ref_frags, data_frags).